lib/web_ui/lib/src/engine/keyboard.dart

```diff
@@ -132,16 +127,18 @@ class Keyboard {
     };
 
     EnginePlatformDispatcher.instance.invokeOnPlatformMessage('flutter/keyevent',
-        _messageCodec.encodeMessage(eventData), _noopCallback);
-  }
-
-  bool _shouldPreventDefault(html.KeyboardEvent event) {
-    switch (event.key) {
-      case 'Tab':
-        return true;
-      default:
-        return false;
-    }
+      _messageCodec.encodeMessage(eventData), (ByteData? data) {
+        if (data == null) {
+          return;
+        }
+        final String response = utf8.decode(data.buffer.asUint8List());
+        final Map<String, dynamic> jsonResponse = json.decode(response);
```

Comment:
I'm curious why not use a `JSONMessageCodec` to decode `data`? There's already one called `_messageCodec` in this file.

Reply:
No good reason, I'm just a goofball.  Converted to use `_messageCodec`, which makes a lot more sense.